packages/lib/src/item-input/command-helpers.ts

```diff
@@ -76,7 +77,7 @@ export const updateFromUserInput = async <T extends object>(command: SmartThings
 			name: 'action',
 			message: 'Choose an action.',
 			choices,
-			default: validationResult === true ? finishAction : editAction,
```

Comment:
When working on unit tests (still in progress), I noticed there is no way to get to this point if `validationResult` is not true.